UIView+MGBadgeView.m

```diff
@@ -42,6 +42,8 @@ - (instancetype)initWithFrame:(CGRect)frame {
 }
 
 - (void)drawRect:(CGRect)rect {
+    [self mg_updateBadgeViewSize];
+    [self mg_updateBadgeViewPosition];
```

Comment:
I don't think the 2 lines above are necessary and I would prefer to not have these calls here.